changelog/add-calypsoify-create-new-post-url

```diff
@@ -0,0 +1,5 @@
+Significance: patch
```

Comment:
It looks like you added the changelog file to the root of the monorepo. Could you remove it, and try something like this instead? That may make things easier.

https://user-images.githubusercontent.com/426388/117431088-1bcef500-af29-11eb-82fd-c89d917dc4f2.mov

Reply:
Done. Thanks for the tip!